assets/scss/components/_slider.scss

```diff
@@ -0,0 +1,92 @@
+.slider {
+	margin-top: 40px;
+
+	.slider2 {
+		position: relative;
+
+		&:before {
+			content: "";
+			position: absolute;
+			width: 1217px;
+			height: 410px;
+			background-image: url(../img/bg.svg);
+		    background-size: cover;
+		    background-repeat: no-repeat;
+		    left: -212px;
+		}
+	}
+
+	.slideshow {
```

Comment:
Be careful stepping outside the BEM methodology.  If this is it's own component, it should live within it's own file.